api/pkg/apis/v1alpha1/utils/apiclient.go

```diff
@@ -560,6 +561,27 @@ func (a *apiClient) PublishActivationEvent(ctx context.Context, event v1alpha2.A
 	return nil
 }
 
+func (a *apiClient) GetActivation(ctx context.Context, activation string, namespace string, user string, password string) (model.ActivationState, error) {
```

Comment:
Add the function in symphony-api.go as well for those that are not token validation.